src/rpc/rawtransaction.cpp

```diff
@@ -210,13 +210,13 @@ static RPCHelpMan getrawtransaction()
                         RPCResult::Type::OBJ, "", "",
                         {
                             {RPCResult::Type::ELISION, "", "Same output as verbosity = 1"},
-                            {RPCResult::Type::NUM, "fee", /*optional=*/true, "transaction fee in " + CURRENCY_UNIT + ", omitted if block undo data is not available"},
+                            {RPCResult::Type::NUM, "fee", "Transaction fee in " + CURRENCY_UNIT},
```

Comment:
The current wording and optionality seems to be correct to me, I [think](https://github.com/bitcoin/bitcoin/blob/79e007d1d60b8d600479d787667d91e05a492ab4/src/core_write.cpp#L254-L258) the fee is only included if undo data is present?

Reply:
The entire block is included only if undo data is present

Reply:
"fee" is not part of the "prevout" block, it is its own, top-level KV pair

Reply:
@dougEfresh this seems unaddressed?